params/config.go

```diff
@@ -852,6 +853,11 @@ func (c *ChainConfig) IsOptimismPreBedrock(num *big.Int) bool {
 	return c.IsOptimism() && !c.IsBedrock(num)
 }
 
+// IsConfigurableMinBaseFeeEnabled returns true if the configurable minBaseFee feature is enabled.
+func (c *ChainConfig) IsConfigurableMinBaseFeeEnabled(time uint64) bool {
```

Comment:
WDYT about dropping the `Enabled` part? We don't have `IsIsthmusEnabled`, only `IsIsthmus`.

Reply:
agreed, done